index.html

```diff
@@ -9,63 +9,51 @@
-    </div>
+    <header>
+        <nav class="main-container">
+        <a href="#" class="logo"> LOGO </a>
+        <a href="#" class="menu">
+            <img src="./images/Union.png" alt="Menu">
+        </a>
+      </nav> 
     </header>
 
-    <section>
-        <div class="main-container">
-            <div class="images">
-
-                <div class="img left-img">
-                    <img src="./images/Rectangle6.png" alt="Rectangle">
-                    <img src="./images/Ellipse11.png" alt="Ellipse">
-                </div>
-
-                <div class="img center-img">
-                    <img src="./images/Rectangle7.png" alt="Rectangle">
-                    <img src="./images/Ellipse5.png" alt="Ellipse">
-                </div>
-
-                <div class="img rigth-img">
-                    <img src="./images/Rectangle4.png" alt="Rectangle">
-                    <img src="./images/Ellipse3.png" alt="Ellipse">
-                </div>
-
-            </div>
-        </div>
-<div class="porfolio-content">
-    <div class="portfolio-h1">
-        <h1>I'm Olamarx </h1>
-        <h1 class="span-portfolio-h1-1">Glad to have</h1>
-        <h1 class="span-portfolio-h1-2">you here!</h1> 
-    </div>
-<div class="portfolio-content-para">
-    <p class="portfolio-para">
-        I am a Web Developer! I can help you build a standard and responsive website. 
-        You can look through some of my handiworks and experience. <span class="portfolio-para-span">Do you like what you see?</span>
-   LET US CONNECT </p>
-</div>
-</div>
-
-
-<p class="connect-with-me">
-        CONNECT WITH ME
-</p>
-
-<div class="contact-link">
-    <div class="social-link">  
-    <a href="#"><i class="fa fas-brands fa-twitter social"></i></a>
-    <a href="#"><i class="fa fas-brands fa-linkedin social"></i></a>
-    <a href="#"><i class="fa fas-brands fa-facebook social"></i></a>
-    <a href="#"><i class="fa fas-brands fa-github social"></i></a>
-</div>
-</div>
-</section>
-
+    <section class="main-header">
+        <section class="main-header-content">
+          <h1 class="main-header-title-content">I am a Web Developer!</h1>
+          <h2 class="main-header-subtitle-content">I can help you build a standard and responsive website.</h2>
+          <p class="main-header-text-description-content">
+             You can look through some of my handiworks and experience.
+          </p>
+          <ul class="social-media-link">
+            <li>
+              <a href="#" class="social-media social-icon">
+                <img src="./images/Vector.jpg" alt="Github"/>
+              </a>
+            </li>
+            <li>
+              <a href="#" class="social-media social-icon">
+                <img src="./images/Linkedinicon.png" alt="Linkedin"/>
+              </a>
+            </li>
+            <li>
+              <a href="#" class="social-media social-icon">
+                <img src="./images/Vector(3).png" alt="Angel"/>
+              </a>
```

Comment:
- [ ] Please make sure that the icons you created are in the same order as the original design 

![image](https://user-images.githubusercontent.com/74744787/158868023-5d770e37-5ea4-41ac-92e7-03b3a60b21ac.png)